docker-java-api/src/main/java/com/github/dockerjava/api/model/Ulimit.java

```diff
@@ -19,16 +23,19 @@
     @JsonProperty("Name")
     private String name;
 
-    @JsonProperty("Soft")
-    private Integer soft;
+    private Long soft;
 
-    @JsonProperty("Hard")
-    private Integer hard;
+    private Long hard;
 
     public Ulimit() {
     }
 
     public Ulimit(String name, int soft, int hard) {
```

Comment:
could you please mark this constructor as deprecated?

Reply:
Done